index.js

```diff
@@ -51,23 +51,45 @@ exports.extend = function(app, options) {
     const mimeTypeLimit = options.mimeTypeLimit ? !Array.isArray(options.mimeTypeLimit) ? [options.mimeTypeLimit] : options.mimeTypeLimit : null;
     delete options.restrictMultiple;
     delete options.mimeTypeLimit;
-    
+
     app.use(busboy(options));
 
     app.use((req, res, next) => {
         var allowUpload = true;
-        
+
         req.body = req.body || {};
         req.files = req.files || {};
 
         if (req.is('json') && req.readable) {
-            jsonBody(req, res, options, (err, body) => {
+            const userProvidedJSONParser = (options.JSON && options.JSON.parse);
+            const subOptions = Object.assign({}, options, {
+                JSON: {
```

Comment:
Are you able to provide `options.JSON` when you initialize busboy?

https://github.com/yahoo/express-busboy#configuration

```js
bb.extend(app, {
    JSON: {
        parse: function(bodyStr, _reviver, done) {
            req.rawBody = bodyStr;

            // If a JSON parser was provided
            // by the user, then let it handle
            // the parsing.
            if (typeof userProvidedJSONParser === 'function')
                return userProvidedJSONParser.call(this, bodyStr, _reviver, done);

            try {
                var result = JSON.parse(bodyStr, _reviver);
                return done.call(this, null, result);
            } catch (error) {
                return done.call(this, error);
            }
        },
    }
});
```

```